### DISCUSSION TITLE
Ask Copilot menu option not visible

### DISCUSSION THREAD

**Discussion by Funkan** [community/community#118535] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have Visual Studio 2022 and CoPilot installed on my desktop, working great and having the option Ask Copilot available on right click on my mouse. I recently bought a new laptop, and have the same installation but no menu option... I can see the "frog" in the bottom left corner, and I get suggestions for code sometimes, but I miss the Ask copilot option. As far as I know, I have everything updated and in order, and it says Copilot is logged in an active.

**Comment by LuciferMorningstar33** [comment]
Try Uninstalling the extension and reinstalling it!

**Reply by LuciferMorningstar33** [reply]
If found helpful pls mark it as a answer

**Comment by devashish2024** [comment]
1. Make sure Copilot is installed
2. Are you logged in?
Try the command platelate and search for copilot or install it. Login, then look if the option is available.

**Comment by Funkan** [comment, marked as answer]
Trying to uninstall it, I realized Copilot was installed, but not the Copilot Chat, and after I installed it, I got Ask Copilot back again :) So no need to uninstall anything.
